### PR TITLE
Use 'create-script' command for all runtime script generators

### PR DESCRIPTION
**Pull request by kichalla**
Currently only Python and Node runtime script generators have the `create-script` and `version` subcommands, With this PR I made change to even make DotNetCore and PHP runtime script generators to have this subcommand. This is so that AppService does not need to do special logic per platform and can have a consistent experience all across.

### CODE REVIEW

File: src/startupscriptgenerator/src/common/commandLineFlags.go [hunk 1]
```diff
@@ -10,34 +10,52 @@ import (
...
-		os.Exit(1)
+	} else {
+		isValidCommand := false
+		for i := 0; i < len(commands); i++ {
+			if suppliedCommandName == commands[i].Name() {
+				isValidCommand = true
+				commands[i].Parse(os.Args[2:])
+				break
+			}
+		}
```
Comment (qianz2) [inline src/startupscriptgenerator/src/common/commandLineFlags.go]: This for loop assumes there is only one valid command, do we guarantee such assumption?

Comment (kichalla) [reply src/startupscriptgenerator/src/common/commandLineFlags.go]: I did not understand your question. Here we try to find the command which is of the same name as what the end user has provided. If we find it, then we break.

Comment (qianz2) [reply src/startupscriptgenerator/src/common/commandLineFlags.go]: Could we avoid for loop and use an existing method to check if the 'commands' array contains 'suppliedCommandName'?

Comment (kichalla) [reply src/startupscriptgenerator/src/common/commandLineFlags.go]: `suppliedCommandName` is a string where as the type of the element in `commands` array is not (for example, I am doing `commands[i].Name()`), so I need to go through each element and do a `.Name()` on that command and compare. Are you asking if there is a C# equivalent in Golang for something like this? `commands.FirstOrDefault(cmd => cmd.Name() ==  suppliedCommandName)`...I checked online and seems like it does not have a built-in way to do this.

Comment (qianz2) [reply src/startupscriptgenerator/src/common/commandLineFlags.go]: Gotcha, thanks for explaining. The annotations are also helpful for us to understand it better.

**Pull request merged**